.travis.yml

```diff
@@ -4,7 +4,6 @@ d:
   - dmd
   - gdc
   - ldc
-  - dmd-2.066.1
-  - dmd-2.070.0-b1
+  - dmd-2.093.1
   - ldc-0.14.0
```

Comment:
Should we update LDC and GDC as well?

Reply:
I can not test gdc and ldc, so I don't have version infos for an update. But yes I can delete the lines with hard-coded version of those compilers.
And no problem with the coding style. I totally agree on keeping consistency.